doc/Learn/Navigation/Advanced/HowTo-UseNavigationView.md

```diff
@@ -234,6 +234,9 @@ Choosing the right control for your navigation needs is important, and one commo
   }
   ```
 
+  > [!NOTE]  
+  > To ensure selecting a `NavigationViewItem` only updates the content region (and doesn’t replace the entire page or the `NavigationView` itself), nest each NavigationView child route under the `Main` route.
+
```

Comment:
```suggestion
  > [!NOTE]  
  > To ensure selecting a `NavigationViewItem` only updates the content region (and doesn’t replace the entire page or the `NavigationView` itself), nest each `NavigationViewItem` as a `RouteMap` under the `Main` `RouteMap`.

```

And maybe add an example, based on the existing `NavigationViewItem` (One, Two, Three) ?